packages/flutter/lib/src/rendering/flex.dart

```diff
@@ -776,8 +776,8 @@ class RenderFlex extends RenderBox with ContainerRenderObjectMixin<RenderBox, Fl
   Rect describeApproximatePaintClip(RenderObject child) => _overflow > 0.0 ? Offset.zero & size : null;
 
   @override
-  String toString() {
-    String header = super.toString();
+  String toShortDescription() {
```

Comment:
for printf debugging I would think seeing the properties would usually be extremely helpful but maybe I'm wrong about that. This CL is already big enough so perhaps in a followup I'll look at all the $this in rendering/.*dart etc.